superset-frontend/package.json

```diff
@@ -149,6 +140,7 @@
     "react-hot-loader": "^4.12.20",
     "react-json-tree": "^0.11.2",
     "react-jsonschema-form": "^1.2.0",
+    "react-loadable": "^5.5.0",
```

Comment:
Previously a dependency of `@superset-ui/chart`, adding it here because it was [moved](https://github.com/apache-superset/superset-ui/pull/774) to `peerDependencies` of `@superset-ui/core`.